### PR TITLE
Add LICENSE and README

### PR DESCRIPTION
**Pull request by brianhlin**
Converting [TestRunsAsVMs](https://twiki.opensciencegrid.org/bin/view/SoftwareTeam/TestRunsAsVMs) into the `README`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+Some quick notes:
+
+-   `universe = vm`
+-   `executable` is just a label
+-   Omit `input`, `output`, and `error`, as they are not used and will cause submit failures
+-   Must select a `vm_type` of `vmware`, `xen`, or `kvm`
+
+### Using VM Universe in CHTC
+
+The only VM universe support in CHTC was using a now-archaic version of [http://www.vmware.com VMware](http://www.vmware.com VMware); this was set up in support of the Thomas Jahns lab. Essentially, in July 2013, there is no current support. However, the CHTC infrastructure team is interested in adding real support for at least kvm.
```
Comment (matyasselmeci) [inline README.md]: The links here did not get translated properly. Should be `[VMware](http://www.vmware.com)`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+
+John H. (11 July 2013) noted that BoxGrinder may run only on the latest Fedora releases, such as 17 and 18; I did not check whether this is true. Also, John noted that he had some problems with BoxGrinder: He could not install both x86-64 and i386 RPMs at the same time, due to limitations in the tool; and there were some issues with EC2, but they would not affect us. John’s BoxGrinder patches are available in his source code repository.
+
+One other thing to think about: BoxGrinder appliance files are fairly simple to understand and reference files contained in the same directory. However, imagefactory uses one giant XML file, with all installed files inlined. Yuck!
+
+Tony Tiradani at Fermi also has experience with creating VM images. He uses Oz, and [has a Subversion repository](https://github.com/holzman/gwms-cloud-vms/tree/development) with an example of his TDL file.
```
Comment (matyasselmeci) [inline README.md]: That does not look like a Subversion repository...

File: README.md [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+guestfish --ro --add vm-io-disk.raw --mount /dev/vda:/ download /osg-test-20130802.log osg-test-20130802.log
+```
+
+### Interactively connecting to a VM
+
+Unfortunately, VM Universe jobs don't have the ssh\_to\_job capacity that's available to other Condor jobs so if we need to investigate test failures in VMU, we'll have to spin up our own VM's. We can do this by taking the images that Neil automatically generates and create new ones from them that don't automatically run osg-test (Right now, Neil has to manually copy over the updated images. Eventually, we should have an automated way to get the most updated VMU images). Here are the steps you need to follow to set up your own VM:
+
+1.  Grab the make-interactive-image from git:
+
+        git clone <git@github.com>:opensciencegrid/vm-test-runs.git
```
Comment (matyasselmeci) [inline README.md]: `<git@github.com>` does not need to be in brackets

File: README.md [hunk 4]
```diff
@@ -0,0 +1,425 @@
...
+----------------------
+
+**Note:** This one time setup must be performed before submitting any test jobs.
+
+1.  Contact Brian L. or Tim C. to have an account created on `osghost.chtc.wisc.edu`
+2.  SSH to `osghost` (you will need to do this from the UW-Madison network) and create a password-less SSH key:
+
+        [user@client ~]$ ssh-keygen -C 'VMU test result upload key for <USER>@osghost' -f ~/.ssh/test_result_upload.key
+
+3.  Contact BrianL or MatS with your public key. They will add your public key to the `authorized_keys` file of the `cndrutil` user on `ingwe`.
```
Comment (matyasselmeci) [inline README.md]: "MatS" looks too much like "Mats". Just remove the "S" - plain "Mat" is what's used elsewhere.

**Comment by matyasselmeci** [comment]
There is a ton of historical information that can probably be removed because it's just confusing, but that should be another PR.

**Comment by brianhlin** [comment]
Fixed all of your issues, can you approve?

**Pull request merged**